packages/formation/sass/base/_va.scss

```diff
@@ -69,8 +69,8 @@ body .row.full {
 
 // Abbr
 
-abbr {
-  border-bottom: 0px !important;
+#content abbr {
```

Comment:
As far as I could tell, the `abbr` tag is always being used within the `#content` wrapper which appears to be sitewide. There is a possibility of this not always being true. Other ways to increase the specificity didn't seen to work though. ie `body abbr { }`